python/ray/util/client/__init__.py

```diff
@@ -30,6 +30,7 @@ def connect(self,
                 metadata: List[Tuple[str, str]] = None,
                 connection_retries: int = 3,
                 namespace: str = None,
+                reconnect_grace_period=None,
```

Comment:
type annotation.

Reply:
No longer relevant since grace period is only through env now